content/docs/projects/approver-policy.md

```diff
@@ -72,13 +72,11 @@ selector.
 least one policy is appropriate for the request but none of those permit the
 request, the request is denied.**
 
-A denied CertificateRequest is considered to be permanently failed. If it was
-created for a Certificate resource, the issuance will be retried with
-[exponential
-backoff](../faq/README.md#what-happens-if-issuance-fails-will-it-be-retried)
-like all other permanent issuance failures. A CertificateRequest that is neither
-approved nor denied (because no matching policy was found) will not be further
-processed by cert-manager until it gets either approved or denied.
```

Comment:
> A CertificateRequest that is neither
> approved nor denied (because no matching policy was found) will not be further
> processed by cert-manager until it gets either approved or denied.

I think that is useful information and suggest we keep it.